dw1000/src/hl/receiving.rs

```diff
@@ -61,168 +56,7 @@ where
     SPI: SpiDevice,
     RECEIVING: Receiving,
 {
-    pub(super) fn start_receiving(&mut self, config: RxConfig) -> Result<(), Error<SPI>> {
-        // Really weird thing about double buffering I can't find anything about.
-        // When a message is received in double buffer mode that should be filtered out,
-        // the radio gives a really short fake interrupt.
-        // This messes up all the logic, so unless a solution can be found we simply don't support it.
-        if RECEIVING::DOUBLE_BUFFERED && config.frame_filtering {
-            return Err(Error::RxConfigFrameFilteringUnsupported);
-        }
```

Comment:
The receiving setup has been moved and is thus hard to review.
Nothing really changed, except for this section which I removed in the PR.

The comment is true. There are useless interrupts, however the driver handles them just fine.
When I put this here 4 years ago, it's likely I was confused. The driver is fine, but my code back then likely wasn't.